### PR TITLE
settings: plugins: Add basic plugin listing

### PR DESCRIPTION
**Pull request by nikosszzz**
* Add /plugins command that shows enabled & disabled plugins, just like Kotlin Aliucord
* Add a (x plugins) field to the /debug command to match Kotlin Aliucord
* Small command description updates
* Update some log messages
* Add log messages for plugin enable & disable

### CODE REVIEW

File: src/api/Commands.ts [hunk 1]
```diff
@@ -84,7 +84,7 @@ export class Commands {
         id: Commands.generateId(),
         type: 1,
         name: "Aliucord",
-        icon: "https://github.com/aliucord.png"
+        icon: "https://avatars.githubusercontent.com/u/78881422?s=200&v=4"
```
Comment (Vendicated) [inline src/api/Commands.ts:87]: ?

File: src/ui/PluginsPage.tsx [hunk 2]
```diff
@@ -102,6 +77,22 @@ function PluginCard({ plugin }: { plugin: PluginManifest; }) {
...
+            description: "Dummy data",
+            version: "1.0.0",
+            authors: [
+                {
+                    username: "Discord",
+                    id: "643945264868098049"
+                }
+            ]
+        };
+        plugins.push(data);
```
Comment (Vendicated) [inline src/ui/PluginsPage.tsx:93]: use Array.map

File: src/api/Commands.ts (hunk 1 above)
Comment (nikosszzz) [reply src/api/Commands.ts:87]: Both don't work from what ive tested, was left there during trying to get the icon to show

File: src/core-plugins/CoreCommands.ts [hunk 3]
```diff
@@ -10,7 +11,7 @@ export default class CoreCommands extends Plugin {
         const ClydeUtils = getByProps("sendBotMessage");
         this.commands.registerCommand({
             name: "echo",
-            description: "Creates Clyde message",
+            description: "Creates Clyde message.",
```
Comment (TymanWasTaken) [inline src/core-plugins/CoreCommands.ts:14]: If you are going to change this, you might as well just fix the grammar too
```suggestion
            description: "Creates a Clyde message.",
```

File: src/core-plugins/CoreCommands.ts [hunk 4]
```diff
@@ -24,9 +25,31 @@ export default class CoreCommands extends Plugin {
             }
         });
 
+        this.commands.registerCommand({
+            name: "plugins",
+            description: "List your Aliucord Plugins.",
```
Comment (TymanWasTaken) [inline src/core-plugins/CoreCommands.ts:30]: More grammar
```suggestion
            description: "Lists all installed Aliucord plugins.",
```

File: src/core-plugins/CoreCommands.ts [hunk 5]
```diff
@@ -24,9 +25,31 @@ export default class CoreCommands extends Plugin {
...
+
+                for (const plugin in aliucord.pluginManager.plugins) {
+                    if (aliucord.pluginManager.isEnabled(plugin)) {
+                        enabledPlugins.push(plugin);
+                    } else {
+                        disabledPlugins.push(plugin);
+                    }
+                }
+
+                const plugins = `**Total plugins**: **${Object.keys(aliucord.pluginManager.plugins).length}**\n\n**Enabled plugins**: **${enabledPlugins.length}**\n> ${enabledPlugins.join(", ") ? enabledPlugins.join(", ") : "None."}\n**Disabled plugins**: **${disabledPlugins.length}**\n> ${disabledPlugins.join(", ") ? disabledPlugins.join(", ") : "None."}`;
```
Comment (TymanWasTaken) [inline src/core-plugins/CoreCommands.ts:44]: I would make this multiline instead of using \n, just to make it more readable
Something like:
```js
const plugins = `
**Total plugins**: ...
**Enabled plugins**: ...
`
```

File: src/core-plugins/CoreCommands.ts [hunk 6]
```diff
@@ -24,9 +25,31 @@ export default class CoreCommands extends Plugin {
...
+
+                ClydeUtils.sendBotMessage(ctx.channel.id, plugins);
+            }
+        });
+
         this.commands.registerCommand({
             name: "eval",
-            description: "Eval javascript",
+            description: "Evaluate JavaScript.",
```
Comment (TymanWasTaken) [inline src/core-plugins/CoreCommands.ts:52]:
```suggestion
            description: "Evaluates raw JavaScript.",
```

File: src/core-plugins/CoreCommands.ts [hunk 7]
```diff
@@ -55,13 +78,13 @@ export default class CoreCommands extends Plugin {
 
         this.commands.registerCommand({
             name: "debug",
-            description: "Posts debug info",
+            description: "Post Debug info.",
```
Comment (TymanWasTaken) [inline src/core-plugins/CoreCommands.ts:81]: Why did you change all these to statements? I would think an action makes more sense

**Review by TymanWasTaken — CHANGES_REQUESTED** [review]

File: src/core-plugins/CoreCommands.ts (hunk 6 above)
Comment (C10udburst) [reply src/core-plugins/CoreCommands.ts:52]: 🤨 why is that an important distinction to make?

Comment (TymanWasTaken) [reply src/core-plugins/CoreCommands.ts:52]: I just added the "raw" because I think it flows better, but you dont have to add that
The main thing is I just don't like the statements for descriptions

**Review by TymanWasTaken — APPROVED** [review]
